docs/sanitation/sanitations.bal

```diff
@@ -0,0 +1,430 @@
+    boolean isODATA4 = false;
+    if (spec.x\-sap\-api\-type=="ODATAV4"){
+        isODATA4 = true;
+    }
+    map<Path> paths = spec.paths;
+    foreach var [key, value] in paths.entries() {
+        if value.get != () {
+            Get getPath = value.get ?: {parameters: []};
+            map<ResponseCode> responses = getPath.responses ?: {};
+            foreach [string,ResponseCode] [_,item] in responses.entries() {
+                if item.description == "Retrieved entities" {
+                    map<ResponseHeader> content = item.content ?: {};
+                    if content.hasKey("application/json"){
+                        ResponseHeader app = content["application/json"] ?: {};
+                        ResponseSchema schema = app.schema ?: {};
+                        if !isODATA4 {
+                            ResponseProperties properties = schema.properties["d"] ?: {properties: ()};
+                            string prefix = properties.title ?: string ``;
+                            prefix =prefix.trim();
+                            if prefix.startsWith("Collection of"){
+                                prefix = prefix.substring(14,prefix.length());
+                                prefix = "CollectionOf"+prefix;
+                            }
+                            if prefix.endsWith("Type"){
+                                prefix = prefix.substring(0,prefix.length()-4);
+                            }
+                            schema.title = prefix+"Wrapper";
+                        }
+                        else {
+                            string schemaTitle = schema.title?: string ``;
+                            if schemaTitle.startsWith("Collection of"){
+                                schema.title = "CollectionOf"+schemaTitle.substring(14,schemaTitle.length()-5);
+                            }
+                        }
+                    }
+                }
+                else if item.description == "Retrieved entity" {
+                    map<ResponseHeader> content = item.content ?: {};
+                    if content.hasKey("application/json"){
+                        ResponseHeader app = content["application/json"] ?: {};
+                        ResponseSchema schema = app.schema ?: {};
+                        string schemaTitle = schema.title ?:string `` ;
+                        if schemaTitle.endsWith("Type"){
+                            schema.title = schemaTitle.substring(0,schemaTitle.length()-4);
+                        }
+                    } 
+                }
+            }
+        }
+        if value.post != () {
+            Post postPath = value.post ?: {parameters: []};
+            map<ResponseCode> responses = postPath.responses ?: {};
+            foreach [string,ResponseCode] [_,item] in responses.entries() {
+                if item.description == "Created entity" || item.description == "Success"{
```

Comment:
Do we need this check. We checked this for get method as based on the description logic changes. AFAIK not here.